src/server/svr_func.c

```diff
@@ -7193,3 +7193,38 @@ action_opt_bf_fuzzy(attribute *pattr, void *pobj, int actmode)
 
 	return PBSE_NONE;
 }
+
+/**
+ * @brief
+ * 		sets the given value for an attribute of any object from server
+ *
+ * @param[in]	pattr	-	pointer to attribute being set
+ * @param[in]	pobj	-	pointer to attribute definition
+ * @param[in]	value	-	value to be set
+ *
+ * @return	void
+ *
+ * @par MT-Safe: No
+ * @par Side Effects: None
+ *
+ */
+void
+set_attr_svr(attribute *pattr, attribute_def *pdef, char *value)
```

Comment:
We already have functions named set_attribute, mgr_set_attr, set_attr, and set_attr_resc. The name set_attr_svr is innocuous, but not very descriptive. Besides the changes proposed in this PR, aren't there dozens of places where this should be called as opposed to the "old way" of setting server attributes?

If you're going to introduce a new function, I suggest you look for all the places it could be applied and refactor accordingly.

Reply:
@mike0042,
I named the function as set_attr_svr because we can read it like "setting an attribute of any object from server" thinking that the older style of setting attributes is mostly used in server and hence named the function like that. Please suggest me a better name if you already have something in your mind. I am fine to change it.

As I mentioned earlier since I am going to log a separate infrastructure RFE to correct the code  where the older style is used(apparently it seems there are several places like that) to newer style we will revisit this function at that and modify it accordingly if required.

Reply:
I don't have a problem with the name you chose, it's just that we already have several functions with similar names. Every time we add another it makes it more difficult to determine which is the correct call to use in a given scenario. Please link the refactoring ticket to PP-746 so we can tell they are related.

Reply:
Done